hivemind/optim/experimental/optimizer.py

```diff
@@ -30,92 +30,101 @@
 
 class Optimizer(torch.optim.Optimizer):
     """
-    Hivemind Optimizer wraps your regular PyTorch Optimizer for training collaboratively with peers.
-    By default, Optimizer is configured to be exactly **equivalent to synchronous training** with target_batch_size;
+    hivemind.Optimizer wraps your regular PyTorch Optimizer for training collaboratively with peers.
+
+    By default, Optimizer is configured to be exactly **equivalent to synchronous training** with target_batch_size.
     There are advanced options make training semi-asynchronous (delay_optimizer_step and delay_gradient_averaging)
-    or even fully asynchronous (local_updates=True). However, these options require careful tuning.
+    or even fully asynchronous (use_local_updates=True).
 
-    :example: The Optimizer can be used as a drop-in replacement for your regular PyTorch Optimizer:
+    :example: The Optimizer can be used as a drop-in replacement for a regular PyTorch Optimizer:
 
     >>> model = transformers.AutoModel("albert-xxlarge-v2")
     >>> dht = hivemind.DHT(initial_peers=INITIAL_PEERS, start=True)
-    >>> opt = hivemind.Optimizer(dht, run_id="run_42", optimizer=lambda params: torch.optim.Adam(params, ...),
-                                 params=model.parameters(), target_batch_size=4096, batch_size_per_step=4)
-    >>> # alternative: opt = hivemind.Optimizer(dht, run_id="run_42", optimizer=torch.optim.Adam(model.parameters())
+    >>> opt = hivemind.Optimizer(dht=dht, run_id="run_42", params=model.parameters(),
+    >>>                          optimizer=lambda params: torch.optim.Adam(params, **other_options),
+    >>>                          target_batch_size=4096, batch_size_per_step=4)
```

Comment:
Let's use the same order of parameters in hivemind.Optimizer definition, both examples in the API docs and the quickstart. It makes it easier to compare the values of the same arguments in these examples, as well as go through them to check whether you've specified all you need in your own code.

Reply:
fixed